src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/InntektsendringerService.kt

```diff
@@ -13,35 +13,55 @@ class InntektsendringerService(
+            nyesteRegistrerteInntekt,
+            forventetInntektForPerson.personIdent,
+            forventetInntektForPerson.forventetInntektForrigeMåned,
+        )
+
+        return EndretInntekt(
+            antallProsentForrigeMåned >= 10 && antallProsentToMånederTilbake >= 10,
+            (antallProsentForrigeMåned + antallProsentToMånederTilbake) / 2,
+        )
     }
 
-    private fun har10ProsentHøyereInntektEnnForventet(nyesteRegistrerteInntekt: List<InntektVersjon>?, ident: String, forventetInntekt: Int?): Boolean {
+    private fun antallProsentHøyereInntektEnnForventet(nyesteRegistrerteInntekt: List<InntektVersjon>?, ident: String, forventetInntekt: Int?): Int {
         if (forventetInntekt == null || nyesteRegistrerteInntekt?.maxOfOrNull { it.versjon } == null) {
             secureLogger.warn("Ingen gjeldende inntekt funnet på person $ident har personen løpende stønad?")
-            return false
+            return 0
         }
-        if (forventetInntekt > 585000) return false // Ignorer alle med over 585000 i årsinntekt, da de har 0 i utbetaling.
+        if (forventetInntekt > 585000) return 0 // Ignorer alle med over 585000 i årsinntekt, da de har 0 i utbetaling.
         val månedligForventetInntekt = (forventetInntekt / 12)
 
         val orgNrToNyesteVersjonMap = nyesteRegistrerteInntekt.associate { it.opplysningspliktig to it.versjon }
-        val inntektListe = nyesteRegistrerteInntekt.filter { it.versjon == orgNrToNyesteVersjonMap.get(it.opplysningspliktig) && it.arbeidsInntektInformasjon.inntektListe != null }.flatMap { it.arbeidsInntektInformasjon.inntektListe!! }
-
+        val inntektListe = nyesteRegistrerteInntekt.filter {
+            it.versjon == orgNrToNyesteVersjonMap[it.opplysningspliktig] && it.arbeidsInntektInformasjon.inntektListe != null
+        }.flatMap { it.arbeidsInntektInformasjon.inntektListe!! }
         val samletInntekt = inntektListe.filterNot {
             ignorerteYtelserOgUtbetalinger.contains(it.beskrivelse) ||
                 (it.inntektType == InntektType.YTELSE_FRA_OFFENTLIGE && it.tilleggsinformasjon?.tilleggsinformasjonDetaljer?.detaljerType == "ETTERBETALINGSPERIODE")
         }.sumOf { it.beløp }
-        if (samletInntekt < halvtGrunnbeløpMånedlig) return false
+
+        if (samletInntekt < halvtGrunnbeløpMånedlig) return 0
+        if (månedligForventetInntekt == 0) return 100 // Prioriterer personer registrert med uredusert stønad, men har samlet inntekt over 1/2 G
+
         secureLogger.info("Samlet inntekt: $samletInntekt - månedlig forventet inntekt: $månedligForventetInntekt  (årlig: $forventetInntekt) for person $ident")
-        return samletInntekt >= (månedligForventetInntekt * 1.1) && samletInntekt > 0 // Må sjekke om samletInntekt er større enn 0 for å ikke få true dersom alle variabler er 0 (antakelig kun i test)
+        val antallProsentInntektEndret = (((samletInntekt / månedligForventetInntekt.toDouble()) * 100) - 100).toInt()
```

Comment:
Hvorfor vil man ikke ha med desimalene her? Evt. rundet av hvis det blir mange. F.eks. om noen har 10,2% endring vil dette bli 10, hvor de ikke trenger å si ifra men de har jo over 10 så burde si ifra. De vil jo ikke prioriteres av denne grunn uansett kanskje

Reply:
Formelen for å beregne endring i prosent er:

`((ny verdi – opprinnelig verdi) / opprinnelig verdi) * 100 % = endring i prosent`